pages/cart.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Compact Store | Cart</title>
+    <script src="https://kit.fontawesome.com/4f4010c8fe.js" crossorigin="anonymous"></script>
+    <link rel="shortcut icon" href="../assets/logo/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="https://compact-ui.netlify.app/stylesheets/style.css">
+    <link rel="stylesheet" href="../stylesheets/style.css">

```

Comment:
I believe you are using your component library on all the pages. So you can import this directly to style.css. This way you won't be repeating this line over and over again.

Reply:
Thanks for the suggestion. Will do this.